ledger/src/blockstore_meta.rs

```diff
@@ -246,7 +274,16 @@ impl ErasureMeta {
 
     pub(crate) fn coding_shreds_indices(&self) -> Range<u64> {
         let num_coding = self.config.num_coding() as u64;
-        self.set_index..self.set_index + num_coding
+        // first_coding_index == 0 may imply that the field is not populated.
+        // self.set_index to be backward compatible.
+        // TODO remove this once cluster is upgraded to always populate
```

Comment:
Might be good to create an issue as a reminder to track this TODO

Reply:
Yeah, already have draft follow up changes.
There is no harm in keeping this code as is, until those upcoming changes are ready to go.
I just need to merge this commit in an earlier release to avoid any compatibility issues.